### PR TITLE
Fix website bug with widget on mobile

### LINKED ISSUE #721: Website, deactivate Discord widget on mobile
On our website, the Discord widget does not properly work on mobile:

Doesnt look good:
first

Goes beyond the size:
second

It even deactivates the animations of parts that come after:
third

Google Search Console also detects flaws with it:
gsc

All in all, we should simply deactivate the whole widget on mobile (but keep it on Desktop).

### PR DESCRIPTION
**Pull request by AthithyaIthayan**
Closes and fixes #721, the problem was that the size for the widget was hardcoded. Now its made relative `100%` and by that scales properly.

### CODE REVIEW

File: application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java [hunk 1]
```diff
@@ -54,20 +56,30 @@ private boolean messageHasNoMediaAttached(Message message) {
...
+
         MessageEmbed originalMessageEmbed =
                 new EmbedBuilder().setDescription(originalMessageContent)
                     .setColor(Color.ORANGE)
                     .build();
 
-        MessageCreateData dmMessage = new MessageCreateBuilder().setContent(
-                "Hey there, you posted a message without media (image, video, link) in a media-only channel. Please see the description of the channel for details and then repost with media attached, thanks 😀")
+        return new MessageCreateBuilder().setContent(message.getAuthor().getAsMention()
+                + " Hey there, you posted a message without media (image, video, link) in a media-only channel. Please see the description of the channel for details and then repost with media attached, thanks 😀")
```
Comment (Zabuzard) [inline application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java:68]: please update ur base branch correctly. ur PR contains unrelated changes from ur other PR.

ur supposed to create different independent branches to avoid such stuff

**Review by Zabuzard — CHANGES_REQUESTED** [review]
the actual changes look good, thanks.

but u gotta fix the issue with the unrelated changes first

File: application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java (hunk 1 above)
Comment (AthithyaIthayan) [reply application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java:68]: means I did fork it I need to sync my fork?

Comment (Zabuzard) [reply application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java:68]: @AthithyaIthayan no. ur PR is from ur `develop` to our `develop`. instead, u should locally create a branch `bugfix/website_widget` based on our `develop` (which doesnt include ur other changes yet). and then have the PR from that branch to our develop.

that said, since u now have contributor-permission, u can actually also work on a branch directly on our repo instead of working with a fork.

**Comment by Taz03** [comment]
can u show how it looks now?

**Comment by AthithyaIthayan** [comment]
image
this is in web 
image
this is over phone

File: website/src/main/resources/public/index.html [hunk 2]
```diff
@@ -74,7 +74,7 @@ <h1><span class="image right"><img style="width: 50%" src="images/logo_frame.png
                 Lots of friendly volunteers to help you learn Java or chat about everyday
                 things.</small></p>
             <ul>
-                <li>20.000 members</li>
+                <li>24.000 members</li>
```
Comment (Zabuzard) [inline website/src/main/resources/public/index.html:77]: please write 25k or leave it at 20k, thanks.
its too specific otherwise, looks odd

**Review by Zabuzard — CHANGES_REQUESTED** [review]

File: website/src/main/resources/public/index.html (hunk 2 above)
Comment (AthithyaIthayan) [reply website/src/main/resources/public/index.html]: Did this

File: application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java (hunk 1 above)
Comment (Zabuzard) [reply application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java:68]: whatever u do, ensure that the PR only has changes for this feature and not the other
